Documentation/guides/building-apps/build-properties.md

```diff
@@ -469,6 +469,15 @@ Support for this property was added in Xamarin.Android 5.1.
 
 This property is `False` by default.
 
+## AndroidEnableObsoleteOverrideInheritance
```

Comment:
Do we need a .NET 8 section for this doc?

https://github.com/xamarin/xamarin-android/blob/main/Documentation/guides/OneDotNetBindingProjects.md#migration-considerations

Reply:
Good question. We do not expect ~anyone to actually use this, however there was an internal customer with a very custom setup that hit this and we did need to provide an opt-out.  Thus we do not intend to widely document or suggest its usage.